data-table/table.js

```diff
@@ -157,6 +157,42 @@ function addColumns(numberOfColumns, index) {
      */
 }
 
+/**
+ * This function takes the id of the list container for the column names
+ * It then adds to the list another space for another column at the bottom of the list
+ * @param config id of the table
+ */
+function addColumn(config) {
+    let container = document.getElementById(config.tableIds.entryBoxDivId);
+    let child = document.createElement("DIV");
+
+    child.innerHTML = "Enter "+ config.datumConfig.names + ":";
+    container.appendChild(child);
+}
+
+/**
+ * This function enables the user to enter the name of the column,
+ * When the container with the id is clicked, an input field is created that enables the user
+ * to input the name of the column.
+ * If the user focuses out of the input field, the name is updated appropriately, and the
+ * input element removed
+ * @param config the id of the table
+ */
+
+function enterColumnValue(config) {
+    let entryBoxDiv = document.getElementById(config.tableIds.entryBoxDivId);
+    let input = document.createElement("INPUT");
+    input.type = 'text';
+    input.placeholder = "Enter " + config.datumConfig.names;
+    entryBoxDiv.innerHTML.innerHTML = input;
```

Comment:
As far as I'm aware, the `.innerHTML` property accesses all the HTML within an element - there is no `.innerHTML` property within that `element.innerHTML`. Kinda like having an empty box and saying, "I want to fill the empty space inside the empty space inside the box." Cleaner to just say, "I want to fill the empty space inside the box". 

Also, as written, if you call this I think it'll overwrite anything else in the entryBoxDivId which we don't want. Use `oldElement.appendChild(newElement)` to safely add new elements to an old one.

Reply:
You are right, in was just my misunderstanding of this.  I will make appropriate changes.  Thanks for your feedback!